src/main/java/org/opentripplanner/standalone/config/routerequest/RouteRequestConfig.java

```diff
@@ -159,6 +152,12 @@ latest arrival time (LAT - EAT).
         )
         .asDuration(dft.searchWindow())
     );
+    vehicleParking.setUseAvailabilityInformation(
+      c
+        .of("useVehicleParkingAvailabilityInformation")
+        .since(V2_1)
```

Comment:
summary?

Reply:
Added in 22fedff9dcdd67ed49d01aca062add859626cc44